README.md

```diff
@@ -4,20 +4,14 @@
 
 Identify and sanitize sensitive information.
 
-## Features
- - Sanitizes HAR files.
- - Multiple files can be sanitized at the same time.<br>
- - Support for sanitizing SAML requests/responses are in progress.
-
-### Supported Browsers
+## Support
 The following browsers are supported:
- - Google Chrome
- - Firefox
+- Google Chrome
+- Firefox
 
-### Limitations
- - Maximum of 10 files can be sanitized at a time.
- - Each file cannot exceed 50 MB.
- - Only HAR files are supported.
+At the moment, it sanitizes HAR files.<br>
+Multiple files can be sanitized at the same time.<br>
```

Comment:
Should we provide a the max number of supported files here?

Reply:
Yes, have added it.